plugins/catalog-backend/src/ingestion/processors/GitLabDiscoveryProcessor.ts

```diff
@@ -94,7 +94,12 @@ export class GitLabDiscoveryProcessor implements CatalogProcessor {
     };
     for await (const project of projects) {
       result.scanned++;
-      if (!project.archived) {
+      if (
```

Comment:
This got pretty large, perhaps we could flip it around so that it skips the excluded ones instead?

```
if (project.archived) {
  continue
}
...etc
```

Think that should be a bit easier to read

Reply:
yep, I prefer that as well!